src/components/Details.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Detail from './Detail';
+import { fetchExecutive, detailPageSuccess } from '../redux/executives';
+
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Remove the eslint disable line here too and include prop-types in the component.